library/std/src/net/tcp.rs

```diff
@@ -869,7 +869,7 @@ impl TcpListener {
     /// use std::net::{TcpListener, TcpStream};
     ///
     /// fn listen_on(port: u16) -> impl Iterator<Item = TcpStream> {
-    ///     let listener = TcpListener::bind("127.0.0.1:80").unwrap();
+    ///     let listener = TcpListener::bind(format!("127.0.0.1:{port}")).unwrap();
```

Comment:
[ToSocketAddrs](https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html) is implemented for `(&str, u16)`, so `format!` is unnecessary here.

Reply:
Hello @the8472,
Thank you for pointing out.
Removed `format!()` macro call.